### PR TITLE
chore: CDN related changes

### LINKED ISSUE #23875: Make changes to the build to include CDN_URL

### PR DESCRIPTION
**Pull request by SatishGandham**
## Description
Make some necessary non function changes here to reduce conflicts with EE.

### CODE REVIEW

File: app/client/src/index.tsx [hunk 1]
```diff
@@ -19,9 +19,12 @@ import GlobalStyles from "globalStyles";
 // enable autofreeze only in development
 import { setAutoFreeze } from "immer";
 import AppErrorBoundary from "./AppErrorBoundry";
+
+__webpack_public_path__ = window.CDN_URL;
+
+console.log();
```
Comment (hetunandu) [inline app/client/src/index.tsx:25]: missed console.log?

**Review by hetunandu — DISMISSED** [review]

**Review by sum35h — APPROVED** [review]

**Comment by sum35h** [comment]
@hetunandu @sharat87 @somangshu  Please do not merge this PR. This is supposed to be a BE only feature.

**Pull request merged**